tests/unit/data/old/config_out.json

```diff
@@ -20,15 +21,17 @@
           "y": "200",
           "width": 200,
           "height": 100,
-          "uid": "Zone 1"
+          "uid": "Zone 1",
+          "align": "center"
```

Comment:
@e-kolpakov Since this is supposed to represent v1 data, I'm wondering if we should remove the `"align"` setting here, and modify the migration code to add the setting if it's not present.

(The same goes for the `"uid"` setting, but I know you didn't add that.)